crates/worker-proto/proto/nemo/relay/worker/v1/plugin_worker.proto

```
@@ -65,6 +65,7 @@ enum RegistrationSurface {
   MARK_SANITIZE_GUARDRAIL = 30;
   SCOPE_SANITIZE_START_GUARDRAIL = 31;
   SCOPE_SANITIZE_END_GUARDRAIL = 32;
+  CONDITIONAL_MIDDLEWARE_GUARDRAIL = 40;
```

Buf (1.72.0)

[error] Enum value name "CONDITIONAL_MIDDLEWARE_GUARDRAIL" should be prefixed with "REGISTRATION_SURFACE_".

(ENUM_VALUE_PREFIX)